### PR TITLE
Adds a more useful, 0.12+ README example

### PR DESCRIPTION
**Pull request by Gowiem**
## what
* Updates README example

## why
* Easier copy / pasta

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -78,41 +78,33 @@ usage: |-
...
+        "s3:GetObject",
+        "s3:ListBucket",
+        "s3:PutObjectAcl"
+      ]
 
-  resource "aws_iam_user_policy" "default" {
-    name   = "${module.fluentd_user.user_name}"
-    user   = "${module.fluentd_user.user_name}"
-    policy = "${data.aws_iam_policy_document.fluentd_user_policy.json}"
+      resources = [ "arn:aws:s3:::bucket_name/*", "arn:aws:s3:::bucket_name/" ]
```
Comment (SweetOps) [inline README.yaml:106]: I know it's more related to styling, but I think the multiline will be much pretty

Comment (Gowiem) [reply README.yaml]: @SweetOps Done 👍

**Review by SweetOps — APPROVED** [review]

**Pull request merged**